custom/icds_reports/utils/data_pull/bihar_datapull.py

```diff
@@ -0,0 +1,72 @@
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from dateutil import parser
+import csv
+
+from custom.icds_reports.models.aggregate import AwcLocation
+from corehq.apps.reports.view_helpers import get_case_hierarchy
+from corehq.form_processor.interfaces.dbaccessors import LedgerAccessors
+from corehq.apps.products.models import SQLProduct
+from datetime import date, timedelta
+
+
+case_accessor = CaseAccessors('icds-cas')
+ls = LedgerAccessors('icds-cas')
+
+awcs = AwcLocation.objects.filter(aggregation_level=5, state_id='f9b47ea2ee2d8a02acddeeb491d3e175').values('doc_id',
+                                                                                          'awc_site_code',
+                                                                                          'awc_name',
+                                                                                          'supervisor_name')
+x = SQLProduct.by_domain('icds-cas')
+default_value = date(1970, 1, 1)
+
+vaccine_ID_name = [(f.product_id, f.name) for f in x]
+
+vaccine_names = [v[1] for v in vaccine_ID_name]
+
+headers = [
+    'Name of Sector',
+    'Anganwadi Center Name',
+    'Name of Village',
+    'Anganwadi Center Code',
+    'Name',
+    'Phone_Number'
+    'case_type',
+    'has_bank_account',
+    'Bank Account Number'
+    'IFSC COde']
+
+data_rows = [headers]
+
+
+def fetch_case_properties(case, awc):
+
+    return [
+        awc['supervisor_name'],
+        awc['awc_name'],
+        awc['awc_name'],
+        awc['awc_site_code'],
+        case.get_case_property('name'),
+        case.get_case_property('phone_number'),
+        case.get_case_property('case_type'),
+        case.get_case_property('has_bank_account'),
+        case.get_case_property('bank_account_number'),
+        case.get_case_property('ifsc_code')
+    ]
+
+
+for awc in awcs:
+    case_ids = case_accessor.get_open_case_ids_in_domain_by_type('ccs_record',[awc['doc_id']])
+    cases = case_accessor.get_cases(case_ids)
+
+    for case in cases:
+        if case.get_case_property('migration_status') != 'migrated':
```

Comment:
migration_status is the case property of person case. you will have to find the parent of the ccs case which is person case on which you can call this get_Case_property function.
[here](https://github.com/dimagi/commcare-hq/pull/26513/files#diff-719069f558b29ec44991434c4e8cdaa0R125) is how it was finding the parent case of type household. similarly you can find the parent case of type person